src/services/axios.ts

```diff
@@ -1,7 +1,8 @@
 import axios from "axios";
 import Cookies from "js-cookie";
 
-export const domain = import.meta.env.VITE_API_HOST || "http://localhost:3000";
+export const domain =
+  import.meta.env.VITE_API_HOST + "/api/v1/" || "http://localhost:3000/api/v1/";
```

Comment:
```suggestion
export const domain =
  `${import.meta.env.VITE_API_HOST}/api/v1/` || "http://localhost:3000/api/v1/";
```

Also let's verify if we need a `/` after v1

Reply:
This is my .env
![image](https://github.com/Multiwoven/multiwoven-ui/assets/54372016/92c0a476-5e52-4a56-b327-34d597d1e33c)

And this is my axios file
![image](https://github.com/Multiwoven/multiwoven-ui/assets/54372016/78d03f49-663d-4892-aec5-6d5075725df3)

Even if there are extra / everything seems to be working properly,  I went to sources, destinations, models and syncs and all API's are getting called properly.